src/loaders/common/loader_helper.py

```diff
@@ -90,6 +96,8 @@ def _convert_values_to_type(docs: list[dict], key: str, col_type: ColumnType):
 
             values.append(doc[key])
         except KeyError as e:
+            if ignore_missing:
+                continue
```

Comment:
I think this would be cleaner if we got rid of the `KeyError` `except` and instead did something like:
```
if key not in doc:
    if not ignore_missing:
        raise ValueError(...)
    continue   # ignore missing keys
```

In the `try` block above

Reply:
👍

Reply:
:+1: